ssv/README.md

```diff
@@ -0,0 +1,66 @@
+# SSV Protocol
+
+This module presents the SSV protocol for executing the duties in a decentralized way.
+
+## System Model
+
+Our **system model** is composed of a set of replicas $N$ (a committee of operators), out of which at most $f = \lfloor \frac{N-1}{3} \rfloor$ may be faulty. I.e., we are dealing with the optimal resiliency for the **Byzantine Fault Tolerance (BFT)** model.
+
+This **committee** is assigned to a set of **validators**, from whom the duties must be executed. For each validator, the private key is shared in a way that each operator holds a **private share key** and its associated public key. Moreover, each operator holds a **network private and public key** for authentication in the network communication.
```

Comment:
Haven't seen this mentioned in this PR, but seems pretty important for general understanding of the whole system:

- how does the process of grouping Operators into Committees look like ? And then Validators are distributed across those Committees somehow ?
- or is it that Validators are grouped into Committees and then Operators are assigned to serve specific committee(s) ?

The way this section currently describes it is incomplete and kind of confuses me even more:

> This **committee** is assigned to a set of **validators**, from whom the duties must be executed.

Reply:
My current understanding would be:
- User chooses Operators he desires (to operate his shares), this bunch of Operators essentially constitutes a Commitee
- this Committee entity ^ could have already existed in the network (originally created by another User in the past) - in which case we don't need to create a 2nd Committee (and can just bundle 2 or more Validators from independent users together); otherwise (Commitee doesn't exist yet) it's created to serve this 1 User for now

is that about right ?